app/src/App/index.js

```diff
@@ -210,6 +230,13 @@ const EditCard = () => {
     await apiClient.editFlashcard(card);
     window.location.href = "/";
   };
+  const editCardAndAddCard = async (card) => {
+    await apiClient.editFlashcard(card);
+  };
```

Comment:
This function is named `editCardAndAddCard`, but it looks like this function just calls `apiClient.editFlashcard`.  I know (I think?) that the functionality of the `edit and add` feature is to save the new card and clear the input fields for the next card, but where does this clearing out of input fields happen?

Reply:
EDIT- I see you're calling `setFront`, `setBack`, etc. in a wrapper function.  In that case I'd rename `editCardAndAddCard` to just `editCard`, since that's all it really does.  The "adding" is handled by a different part of the codebase.

Reply:
If I rename it to editCard, I will have two editCard functions in the same component. The difference between them is that the first one also have a redirection to the home page function, and the second one just edits the card after which the form is cleared. I can rename the first function to "editCardAndRedirect" and second one "editCard"